ptf/saiacl.py

```diff
@@ -425,19 +423,18 @@ def runTest(self):
 
         try:
             print('#### NO ACL Applied ####')
-            print('#### Sending  ', ROUTER_MAC, '| 00:22:22:22:22:22 | 172.16'
-                  '.10.1 | 192.168.100.100 | SPORT 1000 | @ ptf_intf 2')
+            print('#### Sending  ', ROUTER_MAC, '| 00:22:22:22:22:22 | '
+                  '172.16.10.1 | 192.168.100.100 | SPORT 1000 | @ ptf_intf 2')
             send_packet(self, self.dev_port11, self.pkt)
-            print('#### Expecting 00:11:22:33:44:55 |', ROUTER_MAC, '| 172.16'
-                  '.10.1 | 192.168.100.100 | SPORT 1000 | @ ptf_intf 1')
+            print('#### Expecting 00:11:22:33:44:55 |', ROUTER_MAC, '| '
+                  '172.16.10.1 | 192.168.100.100 | SPORT 1000 | @ ptf_intf 1')
             verify_packets(self, self.exp_pkt, [self.dev_port10])
```

Comment:
Can you add ACL stats and confirm that correct acl entry is hit. There are some cases where ACL hits the catch-all entry and packets are dropped. But we don't know if the correct entry is hit.